doc/source/about.rst

```diff
@@ -82,7 +81,7 @@ This simple example covers one step in a processing chain. Additional informatio
 
 The current list of CLIs can be `found here`_.
 
-.. _found here: improver.cli.html
+.. _found here: https://improver.readthedocs.io/en/latest/improver.cli.html
 
```

Comment:
Suggest avoiding hardcoded link to readthedocs for internal links within the documentation. This makes the link work with an offline copy of the built documentation.
For example, ```The current list of CLIs can be :doc:`found here <improver.cli>\`.```

The other links above to environment YAMLs need to be external github links as the YAMLs aren't included in the documentation build.

Reply:
@tjtg as discussed, I revert to original about.rst